hazelcast/src/test/java/com/hazelcast/cache/CacheConfigTest.java

```diff
@@ -203,6 +199,19 @@ public void defaultCacheTest() {
     }
 
     @Test
```

Comment:
instead of using try/catch block, using `@Test(expected = NullPointerException.class)` can be more simple and useful.